### ISSUE TITLE
New Version

### ISSUE THREAD

**Issue by hudrid** [mihaeu/cosmere#45]
Hello, @mihaeu! Thanks for the cool project :)

Are there any plans to release a new version?

We are trying to use your product, but there are some issues:
1. https://github.com/mihaeu/cosmere/issues/4
2. https://github.com/mihaeu/cosmere/issues/32
3. https://github.com/mihaeu/cosmere/issues/34
4. https://github.com/mihaeu/cosmere/issues/47

**Comment by mihaeu** [comment]
Yeah, I've been meaning to release a new version. Thanks for the push.

#4 might not make it in there, because there's a lot of design decisions to be considered. What I instead would like to do is provide a plugin system so that it's easy to add your own macros/transformers and just tweak it the way you need it.

#32 is high up on my list and I'll try to get it fixed soon. Unfortunately this requires reverse-engineering how Confluence builds their links, as there is no documentation on how the slugs are created.

#34 I can have another look at, but this is one of those nasty examples where I would have to touch the internals of the markdown converter.

**Comment by hudrid** [comment]
🔥🔥🔥 Oh, great news! Waiting! Thanks!
